### ISSUE TITLE
Review Client Functionality: Api Accessors 

### ISSUE THREAD

**Issue by Kevin-K** [ozone-development/ozp-iwc#361]
Largely development has been on internals to optimize performance of IWC since version 1.0.0. No work has been done on simplifying the client's function signature.

Currently to do the simplest data api set you have to do the following:
```
client.data().set("/foo",{entity: "Hello World!"})
```

Points of concern:
```
client.data()
```
While this function call simply returns a reference to the data.api request formatter (no complexity other than 1 function call deep), syntactically it gets tedious to do requests:

```
client.data().set("/foo",{entity: "Hello World!"});
client.data().set("/bar",{entity: "Hello World!"});
client.data().set("/buz",{entity: "Hello World!"});
client.data().get("/foo").then(...);
```

**Should we introduce some property to client in its constructor such chat client.Data = client.data()?** 

This may be pointless, as we could demonstrate that users can simply do `var dataApi = client.data()`

```
var dataApi = client.data();
dataApi.set(''/foo',{entity: "Hello World!"});

vs.

client.data().set("/foo", {entity: "Hello World!"});

vs.

client.Data.set("/foo",{ entity: "Hello World!"});
```

**Comment by Kevin-K** [comment]
Addressed in more detail in #362 

The need to access apis through functions is not necessary on the client level as the accessors are available so long as the client is constructed.

Updating tests to prove so, but for future accessing both the current approach `client.data()` is valid, as well as `client.data` (no function call). All api's can be accessed as an object, functional access remains, it will be deprecated but not removed as to not break current use.

**Comment by Kevin-K** [comment]
Tests completed and this feature was included in the 1.2.0 release, see [iwc website](http://ozone-development.github.io/ozp-iwc/1.2.0) for details on 1.2.0 release
